gpu.Dockerfile

```diff
@@ -29,7 +29,7 @@ RUN mkdir -p /webapp/media /webapp/staticfiles
 
 WORKDIR /webapp
 COPY requirements.txt /webapp/
-RUN pip3 install torch==1.10.2+cu111 -f https://download.pytorch.org/whl/torch_stable.html
+RUN pip3 install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu116
```

Comment:
Can you pin the version to `1.13.1`/`0.13.1`? Also we can remove `torchvision` as it's not used.

Reply:
Sure, It has already been pinned in the `requirements.txt`. I'll add it to the dockerfile too.

Reply:
@JeffreyCA, I made the changes that you requested to this branch. 😄 
Let me know if you need anything else